.configure

```diff
@@ -1,7 +1,7 @@
 {
   "project_name": "WordPress-Android",
   "branch": "trunk",
-  "pinned_hash": "08dfbcacef71687eebd0febaf97258e4f708ea54",
+  "pinned_hash": "578dc25a031df0b463a9718464a6e5e23e7d2ec9",
   "files_to_copy": [
     {
       "file": "android/WPAndroid/gradle.properties",
```

Comment:
The `gradle.properties` is also changed in that mobile-secret's new commit sha, to add the new `jp.sentry.dsn` value.